inc/knowbaseitem.class.php

```diff
@@ -498,6 +498,12 @@ static function addVisibilityRestrict() {
     * @return array
     */
    static public function getVisibilityCriteria($forceall = false) {
+      if (Session::haveRight(self::$rightname, self::KNOWBASEADMIN)) {
+         return [
```

Comment:
Returning an empty array should be enough I guess

Reply:
no, i actually tested :D

Reply:
Oh... OK :)